cmd/mattermost/commands/teamargs.go

```diff
@@ -31,3 +32,19 @@ func getTeamFromTeamArg(a *app.App, teamArg string) *model.Team {
 
 	return team
 }
+
+// Removes duplicates and sorts teams by name
+func removeDuplicatesAndSortTeams(teams []*model.Team) []*model.Team {
```

Comment:
I think this is not the right file for it, it is not related to the team args, is just a helper function. I would place this function after the `searchTeamCmdF` in the `team.go` file. Like others that you can find there (like `removeUserFromTeam`).